test/helpers.js

```diff
@@ -67,126 +67,66 @@ module.exports = {
       assert.equal(this.lifWei2Lif(tokenAccountBalances[1]), balances[1]);
       assert.equal(this.lifWei2Lif(tokenAccountBalances[2]), balances[2]);
       assert.equal(this.lifWei2Lif(tokenAccountBalances[3]), balances[3]);
       assert.equal(this.lifWei2Lif(tokenAccountBalances[4]), balances[4]);
     }
-    if (votes){
-      assert.equal(parseInt(tokenAccountVotes[0]), votes[0]);
-      assert.equal(parseInt(tokenAccountVotes[1]), votes[1]);
-      assert.equal(parseInt(tokenAccountVotes[2]), votes[2]);
-      assert.equal(parseInt(tokenAccountVotes[3]), votes[3]);
-      assert.equal(parseInt(tokenAccountVotes[4]), votes[4]);
-    }
-    if (txsSent){
-      assert.equal(parseInt(tokenAccountTxSent[0]), txsSent[0]);
-      assert.equal(parseInt(tokenAccountTxSent[1]), txsSent[1]);
-      assert.equal(parseInt(tokenAccountTxSent[2]), txsSent[2]);
-      assert.equal(parseInt(tokenAccountTxSent[3]), txsSent[3]);
-      assert.equal(parseInt(tokenAccountTxSent[4]), txsSent[4]);
-    }
-    if (txsReceived){
-      assert.equal(parseInt(tokenAccountTxReceived[0]), txsReceived[0]);
-      assert.equal(parseInt(tokenAccountTxReceived[1]), txsReceived[1]);
-      assert.equal(parseInt(tokenAccountTxReceived[2]), txsReceived[2]);
-      assert.equal(parseInt(tokenAccountTxReceived[3]), txsReceived[3]);
-      assert.equal(parseInt(tokenAccountTxReceived[4]), txsReceived[4]);
-    }
-  },
-
-  checkCrowdsale: async function(crowdsale, etherBalance, tokenPrice) {
-    let debug = this.debug;
-    let crowdsaleEtherBalance = await web3.eth.getBalance(crowdsale.contract.address);
-    let crowdsalePrice = await crowdsale.getPrice();
-
-    debug('Contract Balance:', this.toEther(crowdsaleEtherBalance), 'Ether;', this.toWei(crowdsaleEtherBalance), 'Wei');
-    debug('Token Price:', parseInt(crowdsalePrice));
-
-    if (etherBalance)
-      assert.equal(this.toEther(crowdsaleEtherBalance), etherBalance, "Crowdsale should have the expected ether balance");
-    if (tokenPrice)
-      assert.equal(this.toWei(crowdsalePrice), tokenPrice);
   },
 
-  getProposal: async function(token, id) {
-    var proposal = await token.proposals.call(id);
-    var parsedProposal = {
-      target: proposal[0],
-      id: parseInt(proposal[1]),
-      value: parseInt(proposal[2]),
-      description: proposal[3],
-      status: parseInt(proposal[4]),
-      creationBlock: parseInt(proposal[5]),
-      maxBlock: parseInt(proposal[6]),
-      agePerBlock: parseInt(proposal[7]),
-      votesNeeded: parseInt(proposal[8]),
-      actionData: proposal[9],
-      totalVotes: parseInt(proposal[10])
-    };
-    console.log('['+parsedProposal.id+'] To: '+parsedProposal.target+', Value: '+parsedProposal.value +', MaxBlock: '+parsedProposal.maxBlock+', Desc: '+parsedProposal.description+', Status: '+parsedProposal.status, ', Votes: ',parsedProposal.totalVotes,'/',parsedProposal.votesNeeded);
+  simulateCrowdsale: async function(rate, balances, accounts) {
```

Comment:
why don't we keep this function in test/LifToken.js? We are only using it from there for now. If we need it somewhere else, we'll see :). We might decide to copy/paste it, or to extract smaller parts of it. For me helpers should be for general and more simple stuff, this could go to something like `factories.js` but until then I feel better to have it in the same file (easier to work with it).

Also I'm wondering if we shouldn't try and build a property based test for LifToken...

what do you think @AugustoL?